### PR TITLE
Use bootstrap modal for preview on com_media

### PR DESCRIPTION
**Pull request by dgrammatiko**
#### Scope

Eliminate mootools usage and use bootstrap modal for previewing the images.
Also scripts and stylesheets were called all over the place, now they follow Joomla's best practices
#### B/C

As long as the templates use bootstrap (admin area) there shouldn’t be any problems. Any overrides NEED to be updated!
#### Performance

Should be a bit quicker without mootools eating some of the rendering time, downloading, http requests etc.
#### Testing

Apply the patch and see if everything still works correctly.
Report any problems or any ideas for improvement
#### Preview

screen shot 2015-09-05 at 9 43 15

### CODE REVIEW

File: administrator/components/com_media/views/media/tmpl/default.php [hunk 1]
```diff
@@ -83,4 +127,28 @@
...
+//	'bootstrap.renderModal',
+//	'mediaelement',
+//	array(
+//		'title' => JText::_('COM_MEDIA_VIDEO_PREVIEW'),
+//		'footer' => '<button class="btn" data-dismiss="modal" aria-hidden="true">'
+//			. JText::_("JLIB_HTML_BEHAVIOR_CLOSE") . '</button>'
+//	),
+//	'<div id="videoPlayer"><video class="mejs-player" src="none" /></div>'
+//);
+
```
Comment (dgt41) [inline administrator/components/com_media/views/media/tmpl/default.php]: @wilsonge George this muted part together with the other lines above should be enough for the modal to open in the parent window. Still haven’t found a way to update the video source, need to walk through the API/code of the player

Comment (wilsonge) [inline administrator/components/com_media/views/media/tmpl/default.php]: just do something like

```js
jQuery(document).ready(function($){
	window.parent.document.updateUploader();
	$('.img-preview, .preview').each(function(index, value) {
		$(this).on('click', function(e) {
			window.parent.jQuery('#videoPreview.modal video.mejs-player')[0].player.setSrc($(this).attr('href'));
			window.parent.jQuery('#imagePreview').modal('show');
			return false;
		});
	});
});
```

**Comment by ghazal** [comment]
Looks OK.

File: administrator/language/en-GB/en-GB.com_media.ini [hunk 2]
```diff
@@ -96,5 +97,6 @@ COM_MEDIA_UPLOAD_FILE="Upload file"
 COM_MEDIA_UPLOAD_SUCCESSFUL="Upload Successful"
 COM_MEDIA_UPLOAD="Upload"
 COM_MEDIA_UP="Up"
+COM_MEDIA_VIDEO_PREVIEW="Video Preview"
```
Comment (wilsonge) [inline administrator/language/en-GB/en-GB.com_media.ini]: I guess this string needs to go as well

**Comment by wilsonge** [comment]
Tested successfully<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/7819">issues.joomla.org/joomla-cms/7819</a>.</sub>

**Comment by wilsonge** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/7819">issues.joomla.org/joomla-cms/7819</a>.</sub>

File: administrator/language/en-GB/en-GB.com_media.ini (hunk 2 above)
Comment (dgt41) [inline administrator/language/en-GB/en-GB.com_media.ini]: Actually it’s needed for #7819

Comment (wilsonge) [inline administrator/language/en-GB/en-GB.com_media.ini]: This is #7819. Assuming you mean for #7810 but I thought you'd removed all the code for that from this PR?

**Comment by Kubik-Rubik** [comment]
Thank you @dgt41! Merged with https://github.com/joomla/joomla-cms/commit/7e130b63e4c467ad977c446ffa9feca110a14f48

**Pull request closed without merging**